core/src/avm2/globals.rs

```diff
@@ -806,12 +787,17 @@ pub fn load_player_globals<'gc>(
 /// See that tool, and 'core/src/avm2/globals/README.md', for more details
 const PLAYERGLOBAL: &[u8] = include_bytes!(concat!(env!("OUT_DIR"), "/playerglobal.swf"));
 
+// This defines a function called `make_native_table`
```

Comment:
This comment seems outdated. There's no such a function, but rather `NATIVE_TABLE` is defined.